app/main.py

```diff
@@ -2,0 +1,35 @@
+class CommandNameError(Exception):
+    """CommandNameError"""
+
+
+class DoubleNameError(Exception):
+    """DoubleNameError"""
+
+
+class LanCommandError(Exception):
+    """LaneCommandError"""
+
+
+def copy_file(command: str) -> None:
+    split_command = command.split()
+
+    if len(split_command) < 3:
+        raise LanCommandError(
+            "The command must be"
+            " 'cp input_file_name.txt output_file_name.txt'"
+        )
```

Comment:
It's a good practice to validate user input. However, the error message here might be a bit misleading. It's not that the command must be exactly 'cp input_file_name.txt output_file_name.txt', but rather that it must start with 'cp' and include two file names.